### ISSUE TITLE
WrapObjects attached to PhysicalOffsetFrames cause runtime failure

### ISSUE THREAD

**Issue by tkuchida** [opensim-org/opensim-core#1019]
A Muscle can't wrap around a WrapCylinder if the WrapCylinder is attached to a PhysicalOffsetFrame instead of a Body.

A WrapCylinder can be attached to a PhysicalOffsetFrame (and, with #1005, the WrapCylinder is correctly drawn at the PhysicalOffsetFrame). However, the program crashes when `Manager::integrate()` is called if the WrapCylinder is added to a Thelen2003Muscle's GeometryPath (which I'm doing using an appropriately modified version of the `handlePathWrapping()` "helper method" [here](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Sandbox/ExampleHopperDevice/answers.cpp#L99)).

One issue might be in [GeometryPath::getPointForceDirections()](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Simulation/Model/GeometryPath.cpp#L235):
```cpp
for (i = 0; i < np; i++) {
    PointForceDirection *pfd =
        new PointForceDirection(currentPath[i]->getLocation(),
        *(OpenSim::Body*)&(currentPath[i]->getBody()), Vec3(0));
    rPFDs->append(pfd);
}
```
which apparently assumes the path is a chain of `Body`s rather than `PhysicalFrame`s.

**Comment by chrisdembia** [comment]
Seems like `PointForceDirection` needs to be upgraded to use `PhysicalFrame` instead of `Body`?

**Comment by tkuchida** [comment]
> Seems like `PointForceDirection` needs to be upgraded to use `PhysicalFrame` instead of `Body`?

Looks like it but not sure that will resolve the issue. I'll investigate today…

**Comment by tkuchida** [comment]
I wasn't able to resolve this issue. For now, I'll use `DecorativeGeometry::setTransform()`, but I think we should support attaching WrapObjects to PhysicalOffsetFrames.

**Comment by tkuchida** [comment]
Fixed by #1064
